cli/compose/types/types.go

```diff
@@ -268,74 +326,79 @@ func (u UnitBytes) MarshalYAML() (interface{}, error) {
 	return fmt.Sprintf("%d", u), nil
 }
 
+// MarshalJSON makes UnitBytes implement json.Marshaler
+func (u UnitBytes) MarshalJSON() ([]byte, error) {
+	return []byte(fmt.Sprintf("\"%d\"", u)), nil
```

Comment:
```go
return []byte(fmt.Sprintf(`"%d"`, u)), nil
```

Reply:
@shin- it's gonna print the same as `"\"%d\""`, it's just to not having to escape the `"`

Reply:
oh right sorry I blinked and didn't see the backquotes :man_facepalming: